applications/salk-portal/src/components/ExperimentSidebar.jsx

```diff
@@ -143,7 +160,15 @@ const ExperimentSidebar = ({
         setShrink((prevState) => !prevState)
     };
 
-    const sidebarClass = `${classes.sidebar} scrollbar ${shrink ? `${classes.shrink}` : ``}`;
+    const sidebarClass = [`${classes.sidebar} scrollbar ${shrink ? `${classes.shrink}` : ``}`];
+    const PopulationLabel = ({labelText, dotColor}) => {
+      return (
+        <Typography className='population-label' variant='p'>
```

Comment:
@paularah if you inspect in the console you will notice that the variant isn't working, so please remove it.